modules/gdscript/gdscript_parser.h

```diff
@@ -601,6 +620,33 @@ class GDScriptParser {
 				ERR_FAIL_V_MSG(DataType(), "Reaching unhandled type.");
 			}
 
+			Node *get_source_node() const {
+				switch (type) {
+					case CLASS:
+						return m_class;
+					case CONSTANT:
+						return constant;
+					case FUNCTION:
+						return function;
+					case VARIABLE:
+						return variable;
+					case ENUM:
+						return m_enum;
+					case ENUM_VALUE: {
```

Comment:
Gonna be super picky, but why do these have `{}` if they're just returning? :)

Reply:
Maybe just uniformize the switch case. Either adding `{}` to each `case`, or remove them altogether.

Reply:
ah, thanks! just a formatting mistake from a copy-paste